docs/tutorials/telescope/step_by_step.md

```diff
@@ -0,0 +1,688 @@
+Documentation is contained in the `docs` directory.  
+Currently index pages are written in [RST format (Restructured Text)](https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html), 
+ and content pages are written with [Markdown](https://www.sphinx-doc.org/en/master/usage/markdown.html) for simplicity.
+
+It is possible to build the documentation by using the command:
+```
+cd docs
+make html
+```
+This will output html documentation in the `docs/_build/html` directory and the file `docs_/build/index.html` can be
+ opened in a browser to preview what the documentation will look like.
+ 
+A documentation file with info on the telescope should be added in the `docs/telescopes` directory.  
+This documentation should at least include:  
+ * A short summary on the data source
+ * A summary table, see example below 
+ * Any details on set-up steps that are required to run this telescope
+ * Info on any Airflow connections and variables that are used (see further below) 
+ * The latest schema.
+ 
+ Example of a summary table using `eval_rst` to format the RST table:
+ 
+     ```eval_rst
+    +------------------------------+---------+
+    | Summary                      |         |
+    +==============================+=========+
+    | Average runtime              | 10 min  |
+    +------------------------------+---------+
+    | Average download size        | 500 MB  |
+    +------------------------------+---------+
+    | Harvest Type                 | API     |
+    +------------------------------+---------+
+    | Harvest Frequency            | Monthly |
+    +------------------------------+---------+
+    | Runs on remote worker        | True    |
+    +------------------------------+---------+
+    | Catchup missed runs          | True    |
+    +------------------------------+---------+
+    | Table Write Disposition      | Truncate|
+    +------------------------------+---------+
+    | Update Frequency             | Monthly |
+    +------------------------------+---------+
+    | Credentials Required         | No      |
+    +------------------------------+---------+
+    | Uses Telescope Template      | Snapshot|
+    +------------------------------+---------+
+    | Each shard includes all data | Yes     |
+    +------------------------------+---------+
+    ```
+
+### Including Airflow variable/connection info in documentation
+If a newly developed telescope uses an Airflow connection or variable, this should be explained in the documentation on
+ the telescope.  
+An example of the variable/connection is required as well as an explanation on how the value for this 
+ variable/connection can be obtained.
+
+See for example this info section on the Airflow connection required with the google_books telescope:
+
+---
+## Airflow connections
+Note that all values need to be urlencoded.  
+In the config.yaml file, the following airflow connection is required:  
+
+### sftp_service
```

Comment:
This is just an example of the info section in a documentation file of one of the telescopes (google books in this case), so that airflow connection is required for that telescope.
I tried to make clear it is an example bit by adding it in between the '---' bits, but it does not stand out very well.
Any suggestions on how to make it appear as a block/example?